bootstrap/bootstrap.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+TJBOT_FOLDER=$(dirname "$PWD")
```

Comment:
also, i think we need some more logic here to do a `git clone` if they haven't done it already.. e.g. ideally, we'd like people to be able to bootstrap a tjbot the same way they install node:

```
curl -sL https://raw.githubusercontent.com/ibmtjbot/tjbot/master/bootstrap/bootstrap.sh | sudo -E bash -
```

which means the script probably needs to test if it exists in the filesystem inside a checked out `tjbot` folder, or if not, it needs to `git clone https://github.com/ibmtjbot/tjbot` first.